### PR TITLE
S24

### PR DESCRIPTION
**Pull request by Riaboshtan**

### CODE REVIEW

File: apps/user_profile/models.py [hunk 1]
```diff
@@ -91,12 +93,64 @@ class Child(models.Model):
         blank=False,
         verbose_name='Стать'
     )
+
+    city = models.ForeignKey( to=City, default='')
```
Comment (aodarc) [inline apps/user_profile/models.py]: The FK shouldn't have default value

File: apps/user_profile/templates/add_child.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        <p><textarea id="doc_info" class="form-group" name="doc_info" required ></textarea></p>
+    </div>
+
+
+
+    <h4>Пільги дитини</h4>
+    <div class="form-group">
+        <div><label for="privilege">Назва пільги</label></div>
+        <select id="privilege" name="privilege">
+            <option value="">---</option>
```
Comment (aodarc) [inline apps/user_profile/templates/add_child.html]: you should render it from context
context['privilage'] = Child.PRIVILAGE_CHOICES seem like that

File: apps/user_profile/templates/add_child.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            <option value="1">Інваліди I-шої групи</option>
```
Comment (aodarc) [inline apps/user_profile/templates/add_child.html]: or try render form fields in template

File: apps/user_profile/forms.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+from django import forms
+from apps.user_profile.models import Child
+
+
+class RegisterChildForm(forms.ModelForm):
+        class Meta:
+            model = Child
+            fields = ('__all__')
```
Comment (aodarc) [inline apps/user_profile/forms.py]: exclude = ['crated', 'updated']

**Review by maziar0andriy — APPROVED** [review]

**Review by aodarc — APPROVED** [review]

**Review by aodarc — APPROVED** [review]

**Pull request merged**